ui-tests/tests/notebook.spec.ts

```diff
@@ -55,6 +55,16 @@ const testCellOutputs = async (
   }
 };
 
+/**
+ * Custom waitForApplication for the notebook file browser page
+ */
+export async function treeWaitForApplication({ baseURL }, use, testInfo) {
```

Comment:
Is this needed?

Reply:
No it shouldn't, i had lost track of changes. Thanks for pointing this out